sources/procfs.go

```diff
@@ -296,6 +296,44 @@ func (s *lustreSource) generateMDSMetricTemplates() error {
 	return nil
 }
 
+func (s *lustreSource) generateClientMetricTemplates() error {
+	metricMap := map[string][]lustreHelpStruct{
+		"llite": {
+			{"blocksize", "blocksize_bytes", "Filesystem block size in bytes", s.gaugeMetric},
+			{"checksum_pages", "checksum_pages", "Returns '1' if data checksumming is enabled for the client", s.gaugeMetric},
+			{"default_easize", "default_easize_bytes", "Default Extended Attribute (EA) size in bytes", s.gaugeMetric},
+			{"filesfree", "inodes_free", "The number of inodes (objects) available", s.gaugeMetric},
+			{"filestotal", "inodes_maximum", "The maximum number of inodes (objects) the filesystem can hold", s.gaugeMetric},
+			{"kbytesavail", "kilobytes_available", "Number of kilobytes readily available in the pool", s.gaugeMetric},
+			{"kbytesfree", "kilobytes_free", "Number of kilobytes allocated to the pool", s.gaugeMetric},
+			{"kbytestotal", "kilobytes_capacity", "Capacity of the pool in kilobytes", s.gaugeMetric},
```

Comment:
I've got myself rethinking these metrics now (all the kilobytes_* ones). Should they actually be *_kilobytes instead?